AfricanReligions/events/oldgods_ancestor_worship_events.txt

```diff
@@ -4723,6 +5038,15 @@ character_event = {
 		text = EVTDESC_TOG_7520_BIDA
 		picture = GFX_evt_market_byzantine
 	}
+	desc = {
+		trigger = { has_character_modifier = follower_of_babbaku }
+		text = EVTDESC_TOG_7520_BABBAKU
+		picture = GFX_evt_market_altaic
+	}
+	desc = {
+		trigger = { has_character_modifier = follower_of_magec }
+		text = EVTDESC_TOG_7520_MAGEC
```

Comment:
Did you mean for there to be a picture here?

Reply:
No. I was going to use the default picture for that event, the tribal lands picture.

Reply:
Does that work?  Or do you have to specify that picture here because you specified the picture in the other `desc` blocks?